### PR TITLE
Upgrade to 0.2.6

### PR DESCRIPTION
**Pull request by lks21c**
## Summary

- upgrade to 0.2.6 to support sagemaker manager with 2nd draft.

### CODE REVIEW

**Comment by lks21c** [comment]
## Coverage

```
---------- coverage: platform darwin, python 3.10.6-final-0 ----------
Name                         Stmts   Miss  Cover   Missing
----------------------------------------------------------
baram/airflow_manager.py         8      2    75%   15, 24
baram/async_crawler.py          17      2    88%   21-22
baram/conf_manager.py           13      0   100%
baram/ec2_manager.py            19      5    74%   38, 48, 58, 67-68
baram/ecr_manager.py            10      0   100%
baram/glue_manager.py           70     20    71%   54, 107-108, 190, 213-229, 233
baram/iam_manager.py            16      2    88%   43, 53
baram/kms_manager.py            13      1    92%   36
baram/lambda_manager.py         26     12    54%   31, 43-60
baram/log_manager.py            13      0   100%
baram/poetry_manager.py         13      0   100%
baram/process_manager.py        12      1    92%   23
baram/requests_manager.py       12      0   100%
baram/s3_manager.py            115     14    88%   19-20, 118-120, 138-140, 174-176, 216, 218, 227
baram/sagemaker_manager.py      87     63    28%   15-16, 19-24, 27-34, 37-41, 44-75, 78-79, 92-94, 155-157, 162-164, 167, 173-177, 180-184
----------------------------------------------------------
TOTAL                          444    122    73%

```

File: tests/test_ec2_manager.py [hunk 1]
```diff
@@ -27,23 +27,23 @@ def test_list_subnets(em):
...
-def test_get_subnet_id(em):
-    pprint(em.get_subnet_id(em.get_vpc_id('aflow'), 'AFlow-Pri-Sub-AZ1'))
-    em.get_subnet_id(em.get_vpc_id('aflow'), 'AFlow-Pri-Sub-AZ1')
-
-
-def test_get_ec2_id(em):
-    ec2_id = em.get_ec2_id('bastion-host-1')
-    assert ec2_id
-    print(ec2_id)
+#
```
Comment (Baescott) [inline tests/test_ec2_manager.py:30]: How about merging [this](https://github.com/lks21c/baram/pull/31) after this PR? It suddenly came to mind

Comment (lks21c) [reply tests/test_ec2_manager.py:30]: good to go

**Comment by lks21c** [comment]
@alprinf @Baescott

**Review by Baescott — APPROVED** [review]
LGTM

**Review by alprinf — APPROVED** [review]
LGTM!

**Pull request merged**